src/os/bluestore/BlueStore.cc

```diff
@@ -6859,7 +6861,14 @@ int BlueStore::mkfs()
   dout(10) << " freelist_type " << freelist_type << dendl;
 
   // choose min_alloc_size
-  if (cct->_conf->bluestore_min_alloc_size) {
+  dout(0) << __func__ << " optimal_io_size 0x" << std::hex << optimal_io_size
```

Comment:
dout(0) is too low IMO, suggest to use dout(1 or 5) here and there

Reply:
Sounds good - done, thanks!